comet/manager.py

```diff
@@ -48,13 +47,15 @@ def __init__(self, host, port, noconfig=False):
             Dataset broker host.
         port : int
             Dataset broker port.
+        noconfig : bool
+            Ddeprecated/unused.
         """
-        self.broker = "http://{}:{}".format(host, port)
+        self.broker = f"http://{host}:{port}"
```

Comment:
Ehhh... I really don't like doing this sort of thing without validating the caller's input first.  But maybe it's feature!  And I don't think it needs fixing now.

As it is, this lets me point the manager to any any URL containing a colon.  So, I can get creative when instantiating the manager like:
```
manager = Manager(host="en.wikipedia.org/wiki/2001", port="_A_Space_Odyssey")
```
and end up hitting https://en.wikipedia.org/wiki/2001:_A_Space_Odyssey .  Off-hand I can't come up with a security problem associated with that, but...